app/views/dashboard/_edit-mood.html.erb

```diff
@@ -0,0 +1,25 @@
+<section id = "daily-zen-quote">
+    <blockquote class="blockquote">
+      <p class="display-4 mb-0">"Zen Quote"</p>
+      <footer class="blockquote-footer">Someone</footer>
+    </blockquote>
+</section>
+
+<section id = "dashboard-greeting">
+  <h2>Welcome, <%= current_user.name %></h2>
+</section>
+
+<section id = "edit-mood">
+  <div class="card" style="width: 18rem;">
+    <div class="card-body">
+      <h5 class="card-title">Edit Mood</h5>
+      <p class="card-text">
+        <%= form_tag "/mood/#{current_user.mood_entry.id}", method: :put do %>
```

Comment:
I understand that I'm making additional calls each time I call current_user.#method. However, for the sake of time and MVP, I left as is. If there is additional time this weekend, I hope to make a facade

Reply:
I agree - I think we can put this on a list to refactor later if we have time